qa/rgw/store/sfs/tests/sfs-test.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+# Copyright 2022 SUSE, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+img="s3gw:latest"
```

Comment:
this script possibly won't be a good candidate to be living here, I think. It relies on the s3gw container.

Reply:
file dropped